skbio/io/format/embl.py

```diff
@@ -0,0 +1,1568 @@
+            # constructor = Protein
+            raise EMBLFormatError("There's no protein support for EMBL record")
+
+    if constructor == RNA:
+        return DNA(
+            seq, metadata=md, interval_metadata=imd, **kwargs).transcribe()
+    else:
+        return constructor(
+            seq, metadata=md, interval_metadata=imd, **kwargs)
+
+
+# looks like the genbank _parse_genbank
+def _parse_embls(fh):
+    """Chunck multiple EMBL records by '//', and returns a generator"""
+
+    data_chunks = []
+    for line in _line_generator(fh, skip_blanks=True, strip=False):
+        if line.startswith('//'):
+            yield _parse_single_embl(data_chunks)
+            data_chunks = []
+        else:
+            data_chunks.append(line)
+
+
+def _parse_single_embl(chunks):
+    metadata = {}
+    interval_metadata = None
+    sequence = ''
+
+    # define a section splitter with _embl_yield_section function defined in
+    # this module (return the embl section by embl key). returns generator for
+    # each block with different line type
+    section_splitter = _embl_yield_section(
+        lambda line: _get_embl_section(line),
+        skip_blanks=True,
+        strip=False)
+
+    # process each section, like genbank does
+    for section in section_splitter(chunks):
+        # embl_key is line type (eg ID, AC, ...)
+        embl_key = _get_embl_key(section[0])
+
+        # even section have different keys, (RA, RP, ...), I need to get the
+        # correct section to call the appropriate method (eg REFERENCE)
+        embl_section = _get_embl_section(embl_key)
+
+        # search for a specific method in PARSER_TABLE or set
+        # _embl_parse_section_default
+        parser = _PARSER_TABLE.get(
+            embl_section, _embl_parse_section_default)
+
+        if embl_section == 'FEATURES':
+            # This requires 'ID' line parsed before 'FEATURES', which should
+            # be true and is implicitly checked by the sniffer. This is true
+            # since the first section is parsed by the last else condition
+
+            # partials add arguments to previous defined functions
+            parser = partial(
+                parser, metadata=metadata)
+
+        elif embl_section == "COMMENT":
+            # mantain newlines in comments
+            # partials add arguments to previous defined functions
+            parser = partial(
+                parser, join_delimiter="\n")
+
+        # call function on section
+        parsed = parser(section)
+
+        # reference can appear multiple times
+        if embl_section == 'REFERENCE':
+            # genbank data hasn't CROSS_REFERENCE section, To have a similar
+            # metatadata object, I chose to remove CROSS_REFERENCE from
+            # each single reference and put them in metadata. Since I could
+            # have more references, I need a list of CROSS_REFERENCE, with
+            # None values when CROSS_REFERENCE are not defined
```

Comment:
I think not. In testing examples. I have `LK021130.1:74067..75610:rRNA` which has two references. The first has no cross reference, the second yes. So I need a list with `[None, `<cross_reference>`] to model this case. With a empty value, I couldn't know which referene has cross reference